tests/utils/streams/test_kafka.py

```diff
@@ -33,7 +34,7 @@ def decode(self, value: KafkaPayload) -> int:
 
 class KafkaStreamsTestCase(StreamsTestMixin, TestCase):
 
-    configuration = {"bootstrap.servers": "127.0.0.1"}
```

Comment:
This looks fine to me, I think this just worked incidentally because Kafka was consistently available at this location everywhere we have been running tests.